docs/spfx/web-parts/guidance/simplify-adding-web-parts-with-preconfigured-entries.md

```diff
@@ -37,23 +37,23 @@ One of the properties specified in the web part manifest is the **preconfiguredE
-iconImageUrl            |string          |no      |The icon for the web part to be displayed in the toolbox, represented by an image URL. The image at the URL must be exactly 38x38px. If the **officeFabricIconName** property does not have a value, this property must have a value.|`"iconImageUrl": "https://cdn.contoso.com/weather.png"`
-groupId                 |string          |yes     |This group id should be used to decide which toolbox group the web part should be placed into. The client side framework has reserved group ids for default groups. The developer can pick one of those groups. In that case the group field will be ignored. Alternatively, the developer can pick a completely unique id and a group name. The toolbox will then show the web part in its own group.|`"groupId": "6737645a-4443-4210-a70e-e5e2a219133a"`
-group                   |ILocalizedString|no      |Name of the group in the toolbox in which the web part should be displayed. If no value is provided web part will be displayed in the **Custom** group|`"group": { "default": "Content", "nl-nl": "Inhoud" }`
-properties              |TProperties     |yes     |Key-value pair object with default values for web part properties.|`"properties": { "location": "Redmond", "numberOfDays": 3, "showIcon": true }`
+title                   |ILocalizedString|yes     |The web part title that is displayed in the toolbox.              |`"title": { "default": "Weather", "nl-nl": "Weerbericht" }`
+description             |ILocalizedString|yes     |The web part description that is displayed in the toolbox tooltips.|`"description": { "default": "Shows weather in the given location", "nl-nl": "Toont weerbericht voor de opgegeven locatie" } `
+officeFabricIconFontName|string          |no      |The icon for the web part that is displayed in the toolbox. Its value must be one of the [Office UI Fabric icon names](https://dev.office.com/fabric#/styles/icons). If this property has a value, the **iconImageUrl** property will be ignored.|`"officeFabricIconFontName": "Sunny"`
+iconImageUrl            |string          |no      |The icon for the web part that is displayed in the toolbox and is represented by an image URL. The image at the URL must be exactly 38 x 38 px. If the **officeFabricIconName** property does not have a value, this property must have a value.|`"iconImageUrl": "https://cdn.contoso.com/weather.png"`
+groupId                 |string          |yes     |The group id determines which toolbox group will contain the web part. The SharePoint Framework reserves group ids for default groups. The developer can pick one of those groups. If a group id is specified, then the **group** property will be ignored. Alternatively, the developer can pick a completely unique id and a group name. The toolbox will then show the web part in its own group.|`"groupId": "6737645a-4443-4210-a70e-e5e2a219133a"`
```

Comment:
I'm not sure if I might have changed technical meaning here, but originally the groupID said "In that case the group field will be ignored." I thought this meant if you choose a default groupID then the group property gets ignored. Is that the case?